demo/vl-multiselect.html

```diff
@@ -32,10 +32,10 @@ <h2>Multiselect met meerdere opties</h2>
       </code>
   </pre>
     <div class="demo">
-      <select is="vl-multiselect" multiple>
-        <option value="België">België</option>
-        <option value="Duitsland">Duitsland</option>
-        <option value="Frankrijk">Frankrijk</option>
+      <select id="multiselect" is="vl-multiselect" multiple>
```

Comment:
ik zie geen "multiple" attribuut in de source code terugkomen, dus ik denk niet dat dat voor iets dient ... (overal hier)

Reply:
Geen flauw idee. Check bij @coemans .

Reply:
Dat is een native attribuut om van een input select een input multiselect te maken.

Reply:
dat kan dan toch beter in de vl-multi-select zitten om dat attribuut altijd toe te voegen? beetje dwaas dat ge dat als afnemer bij ne MULTIselect nog ne keer moet doen ...

Reply:
Een native multiselect bestaat niet en het lijkt me niet verstandig om een `<vl-multiselect/>` te introduceren. Héél de wereld maakt multiselects door multiple te zetten op een `<select/>` tag, dus ik stel voor dat wij dat ook doen.

_Hopelijk sprint morgen heel de wereld niet van een berg._

Reply:
Ah je bedoelt dat de aanwezigheid van `is="vl-multiselect"` voldoende moet zijn en dat die automatisch het multiple attribuut zet. Lijkt me een goed idee. Hiervoor zou ik wel een aparte PR en branch voor voorzien, anders zijn we te veel aan't doen in één PR en is het voor afnemers niet duidelijk.

Reply:
Zie #76 ter info, terechte opmerking maar nu niet meer gerelateerd aan deze PR.